pkg/util/detector/detector.go

```diff
@@ -309,7 +309,12 @@ func (d *ResourceDetector) ApplyPolicy(object *unstructured.Unstructured, object
 		return err
 	}
 
-	binding := d.BuildResourceBinding(object, objectKey, policy)
+	labels := map[string]string{
```

Comment:
Variable 'labels' collides with the imported package name.  
How about change it to `policyLabels` or something else?

Same as below in `ApplyClusterPolicy`.

Reply:
Make sense, thanks @RainbowMango